contrib/swig/src/main/scala/edu/cmu/dynet/GRUBuilder.scala

```diff
@@ -0,0 +1,18 @@
+package edu.cmu.dynet
+
+/** Builder method for creating GRUs, as in the C++ code. For its public methods see
+  * [[edu.cmu.dynet.RNNBuilder]].
+  */
+class GRUBuilder private[dynet](private[dynet] val builder: internal.GRUBuilder)
```

Comment:
My preference for this name would be GruBuilder (at least, that's Google Java style). Same for LstmBuilder, etc. Let's ignore for now considering we're going to rethink RNNs though.